agents/rust/mediator/mediation/README.md

```diff
@@ -0,0 +1,48 @@
+> 🚧🚧**WIP**🚧🚧: Integration into aries-vcx in process. See main branch for the MVP http service.
+
+# Aries mediator service (http)
+
+## History
+
+This repo was intended to be a test project to try Rust and Axum features.
+
+Over time it has turned into a ~fully functional aries mediator service supporting the following Aries concepts over ***http*** endpoints.
+
+> **Caveat**: Authentication must be handled at another layer. Integration with [aries-vcx] is intended to enable this.
+
+- [Mediator coordination protocol](https://github.com/hyperledger/aries-rfcs/blob/main/features/0211-route-coordination/README.md)
+- [Core:Routing: Forward Message](https://github.com/hyperledger/aries-rfcs/blob/main/concepts/0094-cross-domain-messaging/README.md#corerouting10forward)
+- [Pickup protocol 2.0](https://github.com/hyperledger/aries-rfcs/tree/main/features/0685-pickup-v2)
+
+[aries-vcx]: https://github.com/hyperledger/aries-vcx
+
+Original readme is available below.
+
+> # axum-test-server
+>
+> A axum server app to test and learn Rust and Axum features.
+>
+> The commits are made in such a way as to demonstrate and try one feature or workflow (refactoring) at a time.  
+> So the git log could be educative.  
+>
+> ## Some Interesting commits
```

Comment:
Feel free to include this somewhere along project plan notes, but probably doesn't belong here.